pkg/porter/cnab.go

```diff
@@ -18,6 +19,7 @@ type CNABProvider interface {
 	Install(arguments cnabprovider.InstallArguments) error
 	Upgrade(arguments cnabprovider.UpgradeArguments) error
 	Uninstall(arguments cnabprovider.UninstallArguments) error
+	List(opts printer.PrintOptions) error
```

Comment:
I'm not sure if this needs to go on the CNAB provider interface. 🤔  The others were there because they were mapping to code that was all duffle implemented stuff, that later we may want to swap out with test stubs.

This is all porter's own code that doesn't need to be put into this interface. I'd be happy moving all the code out of provider/list.go into pkg/porter/list.go and removing this interface declaration.

Reply:
Ah good call; moving out 👍